hazelcast/src/main/java/com/hazelcast/internal/util/phonehome/PhoneHomeParameterCreator.java

```diff
@@ -41,7 +41,7 @@ public PhoneHomeParameterCreator() {
         return parameters;
     }
 
-    public PhoneHomeParameterCreator addParam(String key, String value) {
+    public void addParam(String key, String value) {
```

Comment:
It would be a nice improvement to check if the `parameters` map already contains the new key to be added, and throw an `IllegalArgumentException` in that case. It would help to ensure that the map keys returned by the collectors are unique, to avoid accidentally using the a key for multiple purposes. Please also add a unittest to PhoneHomeParameterCreator covering this check.